storage/gcloud/aio/storage/storage.py

```diff
@@ -228,6 +231,15 @@ async def download_metadata(self, bucket: str, object_name: str, *,
         metadata: Dict[str, Any] = json.loads(data.decode())
         return metadata
 
+    async def download_stream(self, bucket: str, object_name: str, *,
+                       headers: Optional[Dict[str, Any]] = None,
```

Comment:
To make the linter happy, please align the args, eg. such that `headers` starts at the same place as `self`.